Wbooks/Library/LibraryController.swift

```diff
@@ -41,11 +74,29 @@ final class LibraryController: UIViewController, UITableViewDelegate, UITableVie
                 
         let cell = tableView.dequeueReusableCell(withIdentifier: LibraryCell.identifier, for: indexPath) as! LibraryCell
         let book = booksArray[indexPath.row]
-        
+                
         cell.coverImage.image = book.cover
         cell.titleLabel.text = book.title
         cell.authorLabel.text = book.author
         
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, didDeselectRowAt indexPath: IndexPath) {
+        cellTapped()
+    }
+    
+    // MARK: Navigation methods
+    @objc private func notificationsTapped() {
+        // Switch to notifications view
+    }
+    
+    @objc private func searchTapped() {
+        // Switch to search view
+    }
+    
+    @objc private func cellTapped() {
+        // Transition to book detail view
+    }
```

Comment:
I think you can remove the `@objc` annotation here. It's only necessary when you call a method through `#selector(someMethod)`